lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/security/LocalFileBasedAccessControlConfig.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.base.security;
+
+import io.airlift.configuration.Config;
+import io.airlift.configuration.validation.FileExists;
+
+import javax.validation.constraints.NotNull;
+
+import java.io.File;
+
+public class LocalFileBasedAccessControlConfig
+        extends BaseAccessControlConfig
```

Comment:
Can we have a standalone config here ? i.e It doesn’t have to extend `BaseAccessControlConfig`

Reply:
yeah, I was thinking about that but didn't want to duplicate shared configuration. But you are right it is better idea to have isolated configs (actually there will be only one config property duplicated)